src/lib/components/commons/FilterPanel/FilterPanel.types.ts

```diff
@@ -53,6 +53,7 @@ export type IFilterPanelProps<T extends string = string> = {
   displayTexts: Record<T, string>;
   status: any;
   activeFilters: any;
+  setSearchValue: (string: string) => void;
```

Comment:
do not use type as a name of parameter.
```suggestion
  setSearchValue: (text: string) => void;
```

Reply:
ok! fixing it